src/CoreText/CTFontManager.cs

```diff
@@ -44,20 +45,46 @@ namespace CoreText {
 	public enum CTFontManagerScope : uint {
 		None = 0,
 		Process = 1,
+#if !NET
 		[iOS (13,0)][TV (13,0)][Watch (6,0)]
+#else
+		[SupportedOSPlatform ("ios13.0")]
+		[SupportedOSPlatform ("tvos13.0")]
+#endif
 		Persistent = 2,
+#if !NET
 		[NoiOS][NoTV][NoWatch]
+#else
+		[UnsupportedOSPlatform ("tvos")]
+		[UnsupportedOSPlatform ("ios")]
+		[UnsupportedOSPlatform ("maccatalyst")]
+		[UnsupportedOSPlatform ("macos")]
+#endif
 		Session = 3,
```

Comment:
I do not believe it was removed from mac os, and I wrong?

Reply:
Good catch!